modules/mediawiki/manifests/init.pp

```diff
@@ -97,6 +97,7 @@
     $recaptcha_secretkey  = lookup('passwords::recaptcha::secretkey')
     $googlemaps_key       = lookup('passwords::mediawiki::googlemapskey')
     $matomotoken          = lookup('passwords::mediawiki::matomotoken')
+    $googlemaps_key       = lookup('passwords::mediawiki::yandextranslationkey')
```

Comment:
Should be yandextranslation_key

Reply:
Why is it not for google maps? https://github.com/miraheze/puppet/blob/9651400d4f605bfa730e1a445f36003855bcc135/modules/mediawiki/templates/PrivateSettings.php#L9

Reply:
huh? There is already a key called 'googlemaps_key', this is a duplicate, not to mention the name is not correct for the what it'll be doing.